doc/source/getting_started/_notebook_stubs/linear_system.ipynb

```diff
@@ -0,0 +1,613 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Rn(2).element([3.0, 0.0])"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "x = X.element([1, 1, 1])\n",
+    "A_op(x)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`A_op` is a bounded linear operator between Hilbert spaces, so it has an adjoint, given by the transposed matrix:\n",
+    "\n",
+    "$$\n",
+    "\\mathcal{A}^\\ast: \\mathscr{Y} \\to \\mathscr{X} \\\\\n",
+    "\\mathcal{A}^\\ast(y) = A^{\\mathrm{T}} y \\\\\n",
+    "A^{\\mathrm{T}} = \n",
+    "\\begin{pmatrix}\n",
+    "  1 & -1 \\\\\n",
+    "  2 & 0 \\\\\n",
+    "  0 & 1\n",
+    "\\end{pmatrix}\n",
+    "$$\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "MatVecOperator: Rn(2) -> Rn(3)"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "A_op.adjoint"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([[ 1., -1.],\n",
+       "       [ 2.,  0.],\n",
+       "       [ 0.,  1.]])"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "A_op.adjoint.matrix"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It can be evaluated on any element of $\\mathscr{Y}$ - in fact, one can check if the adjoint is correct, in the sense that $\\langle \\mathcal{A}(x), b \\rangle_{\\mathscr{Y}} = \\langle x, \\mathcal{A}^* (b) \\rangle_{\\mathscr{X}}$:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "A_op(x).inner(b) == x.inner(A_op.adjoint(b))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Operators can be added, multiplied by scalars (left and right), composed etc.:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Rn(2).element([6.0, 0.0])"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "A_op_times_two = A_op * 2\n",
```

Comment:
Do left multiplication instead of right. Don't want users to get the wrong habit. 98% of the cases they want left.